docs/workloads/azure/data/architecture/deployment_arch_data_azure.md

```diff
@@ -10,7 +10,13 @@ keywords:
   - activity diagram
 ---
 
-The following activity diagram depicts the steps to create the Core Data Platform, as well as the
-Data Engineering Components:
+The following activity diagrams depicts the steps to create the Core Data Platform, as well as additional
```

Comment:
to be adjusted after https://github.com/amido/stacks-azure-data/pull/112 is merged (contents of this file have been moved to the main architecture file)